app/uk/gov/hmrc/trackingconsentfrontend/assets/package.json

```diff
@@ -59,9 +59,15 @@
     "@testing-library/dom": "6.15.0",
     "@testing-library/jest-dom": "5.1.1",
     "@types/jest": "25.1.4",
-    "@typescript-eslint/eslint-plugin": "2.23.0",
-    "@typescript-eslint/parser": "2.23.0",
+    "@typescript-eslint/eslint-plugin": "^2.23.0",
+    "@typescript-eslint/parser": "^2.23.0",
     "backstopjs": "5.0.4",
+    "eslint": "^7.2.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
```

Comment:
I don't think we need these jsx and react specific dependencies.

Reply:
Yeah, I don't think we need that.  I'll remove it and confirm.

Reply:
There are two different options in the documentation https://www.npmjs.com/package/eslint-config-airbnb-typescript

The "If you don't need React support" section doesn't actually work so I used the react section which does work.  It would be good to fix this because I have also had to add this which I'd rather remove:

```
      "settings": {
        "react": {
          "version": "latest"
        }
      }
```